src/custom/components/AccountDetails/Transaction/index.tsx

```diff
@@ -98,78 +106,99 @@ function ActivitySummary(params: {
   if (!activityData) return null
 
   const { activity, type, summary } = activityData
+  const isOrder = activity && type === ActivityType.ORDER
 
   // Order Summary default object
-  const orderSummary: OrderSummaryType = {
-    from: '🤔',
-    to: '🤔',
-    limitPrice: '🤔',
-    validTo: '🤔',
-  }
-
-  const isOrder = type === ActivityType.ORDER
-
+  let orderSummary: OrderSummaryType
   if (isOrder) {
-    const { inputToken, sellAmount, feeAmount, outputToken, buyAmount, validTo, kind, fulfillmentTime } =
-      activity as Order
+    const order = activity as Order
```

Comment:
do you use `order` somewhere else or what is the point of this?

Reply:
you're reassigning `activity` into `order`

Reply:
Yes, it's close to the `if (isOrder)`, I didn't add this, this was just refactorized. It was dedeclared just a few lines below. This Pr just refactors a bit these parts